templates/email-verification-successful.jinja2

```diff
@@ -0,0 +1,8 @@
+{# SPDX-License-Identifier: LGPL-2.1-or-later -#}
+
+Hello {{ username }},
+
+Your email address has been verified successfully.
+
+Thanks,
+KernelCI SysAdmin
```

Comment:
These email templates look great for a first implementation.  Then as a follow-up, I think it might be good to have a GitHub discussion about their contents and probably ask the people who will be working on the UX Analysis for the new web dashboard to provide some input there with best practices etc.

Reply:
Yes, makes sense. Thanks for pointing it out.
I'll create a GH discussion.